### ISSUE TITLE
Mod commands to clear pools

### ISSUE THREAD

**Issue by junbl** [junbl/moxie-discord-bot#8]
Will let mods clear out all pools on a channel that haven't been rolled in the last `n` days

**Comment by Mysticfil** [comment]
Everybody currently has delete rights on the bot. Is this feature really necessary? I'd say pushing the responsibility towards the person using it would be better than spending time on this.

n Days would also need to be a big number since for example, in the westmarch, we could create faction pools that won't be roll for weeks at a time while we finish the current Arc.

**Priority low**

**Comment by junbl** [comment]
yep this is one that I wanted to have an option for up front in case we started seeing performance issues with looking up pools. it hasn't really been an issue yet, I'm not planning to look too much into this unless it does become a problem, and I don't really want to do it without letting admins configure n for their server regardless
